libs/sdk-ui-kit/src/ChartSorting/MeasureDropdown/MeasureDropdown.tsx

```diff
@@ -0,0 +1,259 @@
+            <span className="select-label">
+                <span>by</span>
+            </span>
+            <div className="measure-sorting-dropdown">
+                {disableDropdown ? (
+                    <>
+                        <BubbleHoverTrigger>
+                            <DropdownButton
+                                className="s-inner-aggregation-disabled-button s-inner-measure-dropdown-button"
+                                value={measureName}
+                                disabled={true}
+                                iconLeft={getMeasureIconClassNameBySelected(
+                                    buttonValue.id,
+                                    enableRenamingMeasureToMetric,
+                                )}
+                            />
+                            <Bubble alignPoints={[{ align: "cr cl" }, { align: "cl cr" }]}>
+                                {intl.formatMessage({ id: bubbleMessage })}
+                            </Bubble>
+                        </BubbleHoverTrigger>
+                    </>
+                ) : (
+                    <Dropdown
+                        closeOnMouseDrag
+                        closeOnParentScroll
+                        renderButton={({ isOpen, toggleDropdown }) => (
+                            <BubbleHoverTrigger>
+                                <div ref={buttonRef}>
+                                    <DropdownButton
+                                        className="s-sort-type-measure-button s-inner-measure-dropdown-button"
+                                        value={measureName}
+                                        isOpen={isOpen}
+                                        disabled={disableDropdown}
+                                        onClick={toggleDropdown}
+                                        iconLeft={getMeasureIconClassNameBySelected(
+                                            buttonValue.id,
+                                            enableRenamingMeasureToMetric,
+                                        )}
+                                    />
+                                </div>
+                                <Bubble alignPoints={[{ align: "cr cl" }, { align: "cl cr" }]}>
+                                    {intl.formatMessage({ id: bubbleMessage })}
+                                </Bubble>
+                            </BubbleHoverTrigger>
+                        )}
+                        renderBody={({ closeDropdown }) => (
+                            <DropdownList
+                                width={width}
+                                items={items}
+                                className="gd-measure-sorting-dropdown-body s-measure-sorting-dropdown-body"
+                                renderItem={({ item }) => {
+                                    // TODO consider adding suffix comparison too
+                                    const isSelected = item.title === buttonValue.title;
```

Comment:
Please remove TODO or add there JIRA ID where it will be resolved.

Reply:
Removed. Added the sequence number check.